src/UKHO.ERPFacade.Common/IO/AzureTableReaderWriter.cs

```diff
@@ -74,6 +81,55 @@ await foreach (var entity in entities)
             return records.FirstOrDefault();
         }
 
+        public async Task UpdateEntity(string traceId,string updateColumn)
+        {
+            TableClient tableClient = GetTableClient(ERP_FACADE_TABLE_NAME);
+            EESEventEntity existingEntity = await GetEntity(traceId);
+
+            if (updateColumn == UPDATE_REQUEST_TIME)
+            {
+                existingEntity.RequestDateTime = DateTime.UtcNow;
+            }
+            else if(updateColumn == UPDATE_RESPONSE_TIME)
+            {
+                existingEntity.ResponseDateTime = DateTime.UtcNow;
+            }
+
+            await tableClient.UpdateEntityAsync(existingEntity, ETag.All, TableUpdateMode.Replace);
+            _logger.LogInformation(EventIds.UpdateEntitySuccessful.ToEventId(), "{updateColumn} is updated in azure table successfully.",updateColumn);
+        }
+
+        public void ValidateEntity()
```

Comment:
I would expect a Validate() method to tell me if validation has failed? What happens (apart from logging) when this occurs?